lib/vagrant-libvirt/action/create_domain.rb

```diff
@@ -270,6 +271,7 @@ def call(env)
           end
 
           env[:ui].info(" -- Graphics Type:     #{@graphics_type}")
+          env[:ui].info(" -- Graphics Websocket: #{@graphics_websocket}")
```

Comment:
Probably should only be displayed if not set to `-1`, otherwise should look to update https://github.com/vagrant-libvirt/vagrant-libvirt/pull/1672/files#diff-5590e6b922ec8d4bccf50817db5e075b6445e9fcdd7ad70a1a3b2b1f6702c1b9R551-R559 to output the port assigned once the VM is started.

Reply:
The commit 78b4cf7864bf36bc8c514a86a058d85b077a44d2 should address your review comment. Let me know if I got you wrong. 

Still figuring out how I can make the tests pass without adding `websocket = '-1'` to all test fixtures. Can you give an example for the test `should execute correctly`?

Reply:
Likely need to update the fixtures as this is changing the default output

Reply:
Done, thanks for your input. Mind taking a look again?